### DISCUSSION TITLE
Change filename template for images taken with the camera

### DISCUSSION THREAD

**Discussion by oktayacikalin** [gsantner/markor#1655] [Q&A] [answered]
I would like to be able to change the default filename template to something like IMG_YYYYMMDD_HHMMSS. Is that possible?

**Comment by gsantner** [comment, marked as answer]
new filename template -->

![Screenshot_20220415-014834](https://user-images.githubusercontent.com/6735650/163494572-83725f7f-45d7-476d-b79f-048636680d32.png)

------

Generally want to change it to similar what you want to have. The way nearly all camera software timestamps the filenames. 

![Screenshot_20220414-144625](https://user-images.githubusercontent.com/6735650/163393367-b4927dc9-ecd7-4479-b1ba-e44f472db403.png)


![grafik](https://user-images.githubusercontent.com/6735650/163393353-8fa70f7e-50ce-4666-ba75-d2ea03ca8564.png)

**Reply by oktayacikalin** [reply]
Looks great! When can a release be expected? :-)

**Reply by gsantner** [reply]
2.9.0 is out

**Reply by oktayacikalin** [reply]
Works great - thank you!
